tests/orderly/test_cli.py

```diff
@@ -0,0 +1,275 @@
+import re
+
+from click.testing import CliRunner
```

Comment:
It's a pity they don't have something that just lets you test the arguments that would make it into the target function, because that's really all you want to check but you have to spend this file checking the side effects...